File: C/lt_01_estseq/lt01_estseq_43.c
#include <stdio.h>

int main() {
  float ana = 1.1, maria = 1.5;
  int anos = 0;

  while (ana <= maria) {
    ana += 0.03;
    maria += 0.02;
    anos++;
  }

  printf("Demorará %d Anos.\n", anos);

  return 0;
}

File: rust/lt01-estseq/lt01-estseq-43.rs
fn main() {
    let mut ana = 110;
    let mut maria = 150;
    let mut i = 0;
    while maria > ana {
        maria += 2;
        ana += 3;
        i += 1;
    }
    println!("Demorarão {} anos.", i);
}
